autologout.module

```diff
@@ -64,8 +70,9 @@ function autologout_timeout_validate($timeout, $max_timeout = NULL) {
 /**
  * Adds a field to user/edit to change that users logout.
  */
-function autologout_form_user_form_alter(&$form, FormStateInterface $form_state) {
-  $user = \Drupal::currentUser();
+function autologout_form_user_form_alter(&$form, $form_state) {
+  $user_timeout = _autologout_get_user_timeout();
+  $user = Drupal::currentUser();
```

Comment:
Can you revert this. Could be cleaned up in a separate commit.
Use `\Drupal` for now instead of just `Drupal`.